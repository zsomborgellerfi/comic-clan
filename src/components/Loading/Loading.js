import React from "react";
import ReactLoading from "react-loading";
import "./Loading.scss";

const Loading = () => (
  <div className={"Loading__container"}>
    <ReactLoading type={"spin"} />
  </div>
);

export default Loading;
