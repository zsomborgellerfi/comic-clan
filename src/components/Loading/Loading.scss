@import '../../styles/variables';

.Loading {
  &__container {
    position: fixed;
    top: 0;
    left: 0;
    height: 100vh;
    width: 100vw;
    display: flex;
    align-items: center;
    justify-content: center;
    background-color: rgba($color: #bbc0c4, $alpha: 0.4);
    z-index: 1;
  }
}
