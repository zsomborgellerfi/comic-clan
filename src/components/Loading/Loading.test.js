import React from "react";
import { shallow } from "enzyme";
import Loading from "./Loading";

describe("Loading", () => {
  const wrapper = shallow(<Loading />);

  it("should render Loading correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
