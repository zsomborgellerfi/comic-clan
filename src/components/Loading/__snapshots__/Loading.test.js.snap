// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render Loading correctly 1`] = `
<div
  className="Loading__container"
>
  <Loading
    color="#fff"
    delay={0}
    height={64}
    type="spin"
    width={64}
  />
</div>
`;
